packages/playground-ui/src/components/ui/elements/entry-list/entry-list-item.tsx

```diff
@@ -3,20 +3,20 @@ import { Column, getColumnTemplate } from './shared';
 
 export function EntryListItem({
   item,
-  selectedItem,
+  selectedItemId,
   onClick,
   children,
   columns,
+  isLoading,
 }: {
   item: any;
-  selectedItem: any | null;
-  onClick?: (score: string) => void;
+  selectedItemId?: string;
+  onClick?: (itemId: string) => void;
   children?: React.ReactNode;
   columns?: Column[];
+  isLoading?: boolean;
 }) {
-  const isSelected = selectedItem && selectedItem?.id === item.id;
-
-  console.log('selectedItem', selectedItem, item, isSelected);
+  const isSelected = selectedItemId === item.id;
```

Comment:
you can pass isSelected from the parent, this will allow to remove the need for `item` and `selectedId` which will make the component more generic

Reply:
will remember of this, for the next iteration